app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+
+def create_person_list(people: list) -> None:
+    person_list = []
+
+    person_list.extend([Person
+                        (person_dict["name"], person_dict["age"])
+                        for person_dict in people])
+
+    for person_dict in people:
+        name = person_dict["name"]
+        wife_name = person_dict.get("wife")
+        husband_name = person_dict.get("husband")
+
+        if wife_name is not None:
```

Comment:
If you write  `if wife_name:` it will be the same . As a result you do not need to put `person_dict.get("wife")` into a variable , same with `husband_name`